imap_processing/cdf/tests/test_cdf_attribute_manager.py

```diff
@@ -1,47 +1,121 @@
 from pathlib import Path
 
+# import pytest
+
 from imap_processing.cdf.cdf_attribute_manager import CdfAttributeManager
 
 
+def test_default_attr_schema():
+    """
+    Test function that covers:
+        _load_default_global_attr_schema
+        _load_default_variable_attr_schema
+    """
+
+    # Test
+
+    # Initialize CdfAttributeManager object which loads in default schema
+    cdf_manager = CdfAttributeManager(Path(__file__).parent.parent / "config")
+
+    # Default global tests
+    # Check false case
+    assert cdf_manager.global_attribute_schema["DOI"]["required"] is False
```

Comment:
It's a very minor thing, but you can also check falseness by saying `assert not [statement]` or check trueness with `assert [statement]`